experimental/instrumentation-fetch/src/instrumentation.ts

```diff
@@ -84,6 +86,12 @@ export class FetchInstrumentation extends BaseInstrumentation {
       initCopy.body = body;
     }
 
+    // add Faro RUM header to the request headers
+    const sessionId = faro.api.getSession()?.id;
+    if (sessionId !== null) {
```

Comment:
The id can only be undefined or string. 
Here the ` strict inequality operator (!==)` checks for `null` only.

To check for both `null` and `undefined` we use the ` inequality operator (!=)` 

```suggestion
    if (sessionId != null) {
```